src/Sentry/DebugImage.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Sentry.Extensibility;
+using Sentry.Internal.Extensions;
+
+namespace Sentry
+{
+    /// <summary>
+    /// The Sentry Debug Meta Images interface.
+    /// </summary>
+    /// <see href="https://develop.sentry.dev/sdk/event-payloads/debugmeta#debug-images"/>
+    public sealed class DebugImage : IJsonSerializable
+    {
+        /// <summary>
+        /// Type of the debug image.
+        /// </summary>
+        public string? Type { get; set; }
+
+        /// <summary>
+        /// Memory address, at which the image is mounted in the virtual address space of the process.
+        /// Should be a string in hex representation prefixed with "0x".
+        /// </summary>
+        public string? ImageAddress { get; set; }
+
+        /// <summary>
+        /// The size of the image in virtual memory.
+        /// If missing, Sentry will assume that the image spans up to the next image, which might lead to invalid stack traces.
+        /// </summary>
+        public long ImageSize { get; set; }
```

Comment:
should this be nullable?

Reply:
since below `writer.WriteNumberIfNotNull("image_size", ImageSize);` implies it can be null